docs/contributing/contributor-guide/index.md

```diff
@@ -94,39 +88,23 @@ ROOTDIR=$(pwd) repo forall -c 'git config commit.template $ROOTDIR/.commit-templ
 
 ## Setup the environment
 
-Create the installation root:
-
-```sh
-mkdir -p ~/.opt/liri
-```
-
 The installation root is the directory where all files will be installed when building the Liri master project from QtCreator.
 
-Create a file with environment variables in case you want to run any Liri program from the terminal.
-Run your favorite text editor, write the following text and save it as `~/lenv`:
+To set your environment variables correctly so you will be able to launch the cross platform apps, run
 
 ```sh
-LIRIDIR=~/.opt/liri
-
-export LD_LIBRARY_PATH=$LIRIDIR/lib:$LD_LIBRARY_PATH
-export XDG_DATA_DIRS=$LIRIDIR/share:/usr/local/share:/usr/share:~/.local/share:~/.local/share/flatpak/exports/share
-export XDG_CONFIG_DIRS=$LIRIDIR/etc/xdg:/etc/xdg
-export QT_PLUGIN_PATH=$LIRIDIR/lib/plugins
-export QML2_IMPORT_PATH=$LIRIDIR/lib/qml:$QML2_IMPORT_PATH
-export PKG_CONFIG_PATH=$LIRIDIR/lib/pkgconfig:$PKG_CONFIG_PATH
-export PATH=$LIRIDIR/bin:$PATH
-#export QT_XCB_GL_INTEGRATION=xcb_egl  # uncomment this if you are not using the NVIDIA proprietary driver
+source env-setup.sh
 ```
 
-Every time you open a new terminal and want to run a Liri program you need to set the environment variables with:
+By default, `.build` inside your repo root will be used as installation root. To change this, issue this instead
 
 ```sh
-source ~/lenv
+LIRIDIR="path/to/directory" source env-setup.sh
```

Comment:
`env-setup.sh` accepts an option `nvidia` argument for nvidia users, it should be mentioned here.